spec/fixtures/templates/rb/cfndsl/sample.rb

```diff
@@ -10,7 +10,8 @@
   Output(:One,FnBase64( Ref("One")))
 
   EC2_Instance(:MyInstance) {
+    InstanceType external_parameters["InstanceType"]
```

Comment:
Can we please have two seperate samples? One with compile time parameters and one without?

Reply:
I separated this out, but did I capture your intent?